### PR TITLE
Add unit tests for webrtc.js

### PR DESCRIPTION
**Pull request by vvmnnnkv**
This for https://github.com/OpenMined/syft.js/issues/48

Some corner cases need to be covered, however, I've got concern that mocking browser's RTC might be wrong direction, as this doesn't actually test that RTC works. Tests are not being executed in browser(s).
E.g. I tried to run "with-grid" example to establish connection between Chrome and FF, and this didn't work. Maybe this should be covered by some separate integration test suite, though, not sure.

### CODE REVIEW

File: test/_webrtc-mocks.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+export class RTCPeerConnectionMock {
+  constructor(options, optional) {
+    this.constructorSpy.apply(this, arguments);
+    this.localDescription = null;
+    this.remoteDescription = null;
+    this.iceCandidates = [];
+    this.sentMessages = [];
+  }
+
+  constructorSpy() {
```
Comment (cereallarceny) [inline test/_webrtc-mocks.js]: Is this method necessary?

File: test/webrtc.test.js [hunk 2]
```diff
@@ -1,33 +1,275 @@
-import { WebSocket } from 'mock-socket';
+import 'regenerator-runtime/runtime';
 
 import {
   WEBRTC_PEER_CONFIG,
   WEBRTC_PEER_OPTIONS,
-  Logger
+  WEBRTC_JOIN_ROOM,
+  Logger, WEBRTC_INTERNAL_MESSAGE, WEBRTC_PEER_LEFT
```
Comment (cereallarceny) [inline test/webrtc.test.js]: Put Logger as the last import so it's in order please :)

File: test/webrtc.test.js [hunk 3]
```diff
@@ -1,33 +1,275 @@
-import { WebSocket } from 'mock-socket';
```
Comment (cereallarceny) [inline test/webrtc.test.js:1]: Do we still need this package for other tests?  If not, remove it from the project.  Otherwise, let's keep it.

**Review by cereallarceny — COMMENTED** [review]
All looks good, a few comments.  In the future, should we look at replacing these tests with Puppeteer?  https://jestjs.io/docs/en/puppeteer

File: test/webrtc.test.js (hunk 3 above)
Comment (vvmnnnkv) [reply test/webrtc.test.js:1]: Required by socket.test.js

File: test/webrtc.test.js (hunk 2 above)
Comment (vvmnnnkv) [reply test/webrtc.test.js]: OK :)

File: test/_webrtc-mocks.js (hunk 1 above)
Comment (vvmnnnkv) [reply test/_webrtc-mocks.js]: Removed that. Tested constructor params in different way.
